README.md

```diff
@@ -22,6 +22,10 @@ It also compiles for Android but never been tested on it.
 
 The minimum-supported version fo `rustc` is **1.45**.
 
+### Use in binaries distributed on the MacOS App Store
+Apple has restrictions as to which APIs can be used in binaries distributed through the app store.
```

Comment:
Please add an empty line before this one.

Reply:
Added.

Reply:
I don't see it. ;)

Reply:
It should be there now, sorry about that. I misread what line this comment was on 🙃